rasa/shared/utils/io.py

```diff
@@ -626,3 +625,21 @@ def is_subdirectory(path: Text, potential_parent_directory: Text) -> bool:
 def random_string(length: int) -> Text:
     """Returns a random string of given length."""
     return "".join(random.choices(string.ascii_uppercase + string.digits, k=length))
+
+
+def handle_print_blocking(output: Text) -> None:
```

Comment:
We must add unit test for this function to make sure that:
* portalocker.Lock is called properly - mock `Lock` function
* AnsiToWin32 is called properly if `sys.platform == "win32"`
* print is called with proper `output`, `file` and `flush` parameters